src/Sulu/Component/Content/Types/ResourceLocator/Strategy/ResourceLocatorGeneratorInterface.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\Content\Types\ResourceLocator\Strategy;
+
+/**
+ * Interface for resource-locator generator.
+ */
+interface ResourceLocatorGeneratorInterface
```

Comment:
Am I right with the assumption that this should be replaced by the `RouteGenerator` from the `RouteBundle` at some point in the future?